ui/desktop/tests/e2e/fixtures/authenticateTest.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef, no-empty-pattern */
+
+const { electronTest } = require('../fixtures/electronTest');
+const LoginPage = require('../pages/loginPage');
+
+exports.authenticateTest = electronTest.extend({
+  // TODO: Should we put these in a config file?
```

Comment:
I think a config file would make it easy for any future reference

Reply:
Yup I think so too. I think once we start building out our config it might make sense to move them over